interfaces/acados_template/acados_template/acados_ocp_options.py

```diff
@@ -324,14 +324,13 @@ def collocation_type(self):
     @property
     def regularize_method(self):
         """Regularization method for the Hessian.
-        String in ('NO_REGULARIZE', 'MIRROR', 'PROJECT', 'PROJECT_REDUC_HESS', 'CONVEXIFY') or :code:`None`.
+        String in ('NO_REGULARIZE', 'MIRROR', 'PROJECT', 'PROJECT_REDUC_HESS', 'CONVEXIFY', 'GERSHGORIN_LEVENBERG_MARQUARDT').
 
         - MIRROR: performs eigenvalue decomposition H = V^T D V and sets D_ii = max(eps, abs(D_ii))
         - PROJECT: performs eigenvalue decomposition H = V^T D V and sets D_ii = max(eps, D_ii)
         - CONVEXIFY: Algorithm 6 from Verschueren2017, https://cdn.syscop.de/publications/Verschueren2017.pdf, does not support nonlinear constraints
         - PROJECT_REDUC_HESS: experimental
-
-        Note: default eps = 1e-4
+        - GERSHGORIN_LEVENBERG_MARQUARDT: estimates the smallest eigenvalue block-wise using Gershgorin circles and adds multiple of identity to the block, such that smallest eigenvalue after regularization is at least reg_epsilon
```

Comment:
maybe simpler:

estimates the smallest eigenvalue of each Hessian block using Gershgorin circles and adds a diagonal matrix to each block such that smallest eigenvalue after regularization is at least reg_epsilon

Reply:
I kept the word identity instead of diagonal matrix, as it is more specific. Otherwise addressed the comment.